pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -133,58 +135,45 @@ func (r *Reconciler) reconcile(ctx context.Context, cronjob *v1alpha1.CronJobSou
 	_, err := cron.ParseStandard(cronjob.Spec.Schedule)
 	if err != nil {
 		cronjob.Status.MarkInvalidSchedule("Invalid", "")
-		return err
+		return fmt.Errorf("invalid schedule: %v", err)
 	}
 	cronjob.Status.MarkSchedule()
 
 	if cronjob.Spec.Sink == nil {
 		cronjob.Status.MarkNoSink("Missing", "Sink missing from spec")
-		return fmt.Errorf("Sink missing from spec")
+		return errors.New("spec.sink missing")
 	}
 
 	sinkObjRef := cronjob.Spec.Sink
 	if sinkObjRef.Namespace == "" {
 		sinkObjRef.Namespace = cronjob.Namespace
 	}
 
-	cronjobDesc := cronjob.Namespace + "/" + cronjob.Name + ", " + cronjob.GroupVersionKind().String()
+	cronjobDesc := fmt.Sprintf("%s/%s,%s", cronjob.Namespace, cronjob.Name, cronjob.GroupVersionKind().String())
 	sinkURI, err := r.sinkReconciler.GetSinkURI(sinkObjRef, cronjob, cronjobDesc)
 	if err != nil {
 		cronjob.Status.MarkNoSink("NotFound", "")
-		return err
+		return fmt.Errorf("getting sink URI: %v", err)
```

Comment:
This should start with `error` or some other word that indicates error. Sometimes error strings are printed with no surrounding indicator that an error occurred.

Alternately, if you're sure this is never printed without an error designation, ignore this comment.

Reply:
I go back and forth on this. Because I am adding this prefix string at every level of the stack, I think I don't want something like 'error foo: error bar: error baz'. I agree that when logged, it needs to say that it is an error.

Concretely, this is logged in Reconcile with the error message 'Error reconciling CrojJobSource' and then this message is put as the value of the 'error' key.